frontend/front/src/pages/Admin/home/CreateNewHome.js

```diff
@@ -1,28 +1,47 @@
-import { Box, Button, TextField, Typography } from "@mui/material";
+import { Alert, Box, Button, TextField, Typography } from "@mui/material";
 import { Controller, useForm } from "react-hook-form";
 
-import React from "react";
+import React, { useEffect } from "react";
 import { useNavigate } from "react-router-dom";
+import { useCreateHomeMutation } from "../../../api/apiSlice";
 
 import { withAdminPrefix, ADMIN_HOME } from "../../../routing/routes";
 
 const CreateNewHome = () => {
   const navigate = useNavigate();
-  const { handleSubmit, control } = useForm({
+  const { handleSubmit, control, reset } = useForm({
     defaultValues: {
       streetNumber: "",
-      address: "",
```

Comment:
The back end uses different casing for these fields (`street_number` instead of `streetNumber` etc.) so I'm not sure that this will work.